importer.py

```diff
@@ -33,6 +32,9 @@ class MissingPathException(Exception):
 class Importer:
 
     def __init__(self, db_config_class, collection_name):
+        # clear_txt(path='import_monitoring.txt')
```

Comment:
my bad another mistake,  should have read over my code more closely. will remove this unneeded code on next push